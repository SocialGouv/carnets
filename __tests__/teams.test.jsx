import { vi, expect, test } from "vitest";
import { render } from "@testing-library/react";
import Component from "../src/components/teams.tsx";

test("Component", async () => {
  const { container } = render(await Component({ selectedTeamSlug: "domifa" }));
  expect(container).toMatchSnapshot();
});
