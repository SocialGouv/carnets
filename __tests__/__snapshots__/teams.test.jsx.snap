// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Component 1`] = `
<div>
  <nav
    aria-labelledby="fr-sidemenu-:r0:-title"
    class="fr-sidemenu fr-sidemenu--sticky"
    id="fr-sidemenu-:r0:"
  >
    <div
      class="fr-sidemenu__inner"
    >
      <button
        aria-controls="fr-sidemenu-:r0:-collapse"
        aria-expanded="false"
        class="fr-sidemenu__btn"
        hidden=""
      >
        Liste des startups
      </button>
      <div
        class="fr-collapse"
        id="fr-sidemenu-:r0:-collapse"
      >
        <div
          class="fr-sidemenu__title"
          id="fr-sidemenu-:r0:-title"
        >
          Les startups de la Fabrique
        </div>
        <ul
          class="fr-sidemenu__list"
        >
          <li
            class="fr-sidemenu__item"
          >
            <a
              aria-current="page"
              class="fr-sidemenu__link"
              href="/posts/domifa"
              target="_self"
            >
              domifa
            </a>
          </li>
        </ul>
      </div>
    </div>
  </nav>
</div>
`;
